public/api/conf/1.0/application.raml

```diff
@@ -93,103 +93,103 @@ documentation:
-#                examples:
-#                  missingHeader:
-#                    description: One or more of the required headers are missing
-#                    value:
-#                      code: MISSING_HEADER
-#                  invalidHeader:
-#                    description: One or more of the header values are invalid
-#                    value:
-#                      code: INVALID_HEADER
-#                  invalidNino:
-#                    description: Invalid NINO format
-#                    value:
-#                      code: INVALID_NINO#
-#
-#          404:
-#            body:
-#              application/json:
-#                type: types.errorResponse
-#                examples:
-#                  resourceNotFound:
-#                    description: The supplied NINO was not found
-#                    value:
-#                      code: RESOURCE_NOT_FOUND
-#
-#          502:
-#            body:
-#              application/json:
-#                type: types.errorResponse
-#                examples:
-#                  resourceNotFound:
-#                    description: Dependent systems are currently not responding
-#                    value:
-#                      code: BAD_GATEWAY
+  /{nino}:
+    uriParameters:
+      nino:
+        description: |
+          National Insurance Number.
+
+          Must conform to the regular expression
+
+          <code>^((?:[ACEHJLMOPRSWXY][A-CEGHJ-NPR-TW-Z]|B[A-CEHJ-NPR-TW-Z]|G[ACEGHJ-NPR-TW-Z]|[KT][A-CEGHJ-MPR-TW-Z]|N[A-CEGHJL-NPR-SW-Z]|Z[A-CEGHJ-NPR-TW-Y])[0-9]{6}[A-D ]?$</code>
+        type: string
+        example: "MZ006526C"
+    /periods:
+      get:
+        displayName: Retrieve all Breathing Space periods for a NINO
+        description: Retrieve all Breathing Space periods for the given NINO.
+        (annotations.scope): "read:breathing-space-periods"
+        securedBy: [ sec.oauth_2_0: { scopes: [ "read:breathing-space-periods" ] } ]
+        headers:
+          Content-Type:
+            description: Specifies the content type of the request payload. Must be same as the example given.
+            type: string
+            required: true
+            example: application/json
+          Correlation-Id:
+            description: A unique UUID for this request
+            type: string
+            required: true
+            example: 24a8a24f-7126-4aa7-b690-259b2eaccaee
+          Request-Type:
+            description: |
+              Context flag for this request. Signifies if this request is the result of a human attended operation or an automated operation.
+
+              The two possible values are ‘DA2_BS_ATTENDED’ or ‘DA2_BS_UNATTENDED’.
+            type: string
+            required: true
+            example: DA2_BS_ATTENDED
+          Staff-Id:
+            description: |
+              The Staff identifier associated with the human attending the operation that generated this request.
+
+              Must be a 7 digit numeric value
```

Comment:
Is it worth to add that for `DA2_BS_UNATTENDED` `Staff-id` must have value `"0000000"` ?

Reply:
Yes, good point. I'll add that in.

Reply:
I've added:
"Please note: when the 'Request-Type' header has the value ‘DA2_BS_UNATTENDED’ then this header's value must equal '0000000' "
We don't check that for ‘DA2_BS_ATTENDED’ that value is not '0000000' do we?

Reply:
I'm removing the UserId header when the value is "0000000". EIS is not expecting the UserId header, and accordingly we need to remove it, for DA2_BS_UNATTENDED. If we allow "0000000" for DA2_BS_ATTENDED this header will be removed and we don't want this of course.